recipes/wildmeshing/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wildmeshing" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wildmeshing/wildmeshing-python/archive/{{ version }}.tar.gz
+  sha256: 12c85ff3ec97d7020bb705dcc76f4a545be13c5537b78ae66edbf4749a6ee56c
+
+build:
+  number: 0
+  skip: True  # [py<31]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+    - libblas
+    - libcblas
+    - numpy
+    - gmp     # [unix]
+    - mpir    # [win]
+  run:
+    - python
+    - mpir    # [win]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - wildmeshing
+
+about:
+  home: https://github.com/wildmeshing/wildmeshing-python
+  license: LGPL
```

Comment:
It looks like `wildmeshing` is under Mozilla Public License Version 2.0? In which case, the SPDX identifier is `MPL-2.0` or `MPL-2.0-no-copyleft-exception`.

Reply:
It should be LGPL because there's a LGPL component vendored